docs/learning-powershell/using-vscode.md

```diff
@@ -73,56 +73,91 @@ you will need to add a new variable to your user settings file.
 
 Debugging with Visual Studio Code
 ----
+### No-workspace debugging
+As of Visual Studio Code version 1.9 you can debug PowerShell scripts without having to open the folder containing the PowerShell script.
+Simply open the PowerShell script file with **File->Open File...**, set a breakpoint on a line (press F9) and then press F5 to start debugging.
+You will see the Debug actions pane appear which allows you to break into the debugger, step, resume and stop debugging.
+
+### Workspace debugging
+Workspace debugging refers to debugging in the context of a folder that you have opened in Visual Studio Code using **Open Folder...** from the **File** menu.
+The folder you open is typically your PowerShell project folder and/or the root of your Git repository.
+
+Even in this mode, you can start debugging the currently selected PowerShell script by simply pressing F5.
+However, workspace debugging allows you to define multiple debug configurations other than just debugging the currently open file.
+For instance, you can add a configurations to:
+
+* Launch Pester tests in the debugger
+* Launch a specific file with arguments in the debugger
+* Launch an interactive session in the debugger
+* Attach the debugger to a PowerShell host process
+
+Follow these steps to create your debug configuration file:
+1. Open the **Debug** view by pressing **Ctrl+Shift+D** (**Cmd+Shift+D** on Mac).
```

Comment:
I've made all the other changes. But I'm not sure I follow this one.  What is the issue?  I have VSCode set up to trim trailing whitespace which is why this probably changed.